lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -856,3 +856,131 @@ def test_streamplot():
     ax.streamplot(x, y, u, v, transform=ccrs.PlateCarree(),
                   density=(1.5, 2), color=mag, linewidth=2 * mag)
     return fig
+
+
+@pytest.mark.natural_earth
+@pytest.mark.mpl_image_compare(filename='annotate_mercator.png', style='mpl20')
```

Comment:
```suggestion
@pytest.mark.mpl_image_compare(filename='annotate_mercator.png')
```
Do we need the old style for some reason here? Since this is a new test, I'd say just stick with the defaults.

Reply:
Ah, I had just copied the test above, have removed that in the process of consolidating the tests.